aiida/cmdline/commands/cmd_computer.py

```diff
@@ -237,7 +238,12 @@ def computer_setup(ctx, non_interactive, **kwargs):
         echo.echo_success(f'Computer<{computer.pk}> {computer.label} created')
 
     echo.echo_report('Note: before the computer can be used, it has to be configured with the command:')
-    echo.echo_report(f'  verdi computer configure {computer.transport_type} {computer.label}')
+
+    profile = ctx.obj['profile']
+    if profile.name == get_default_profile():
+        echo.echo_report(f'  verdi computer configure {computer.transport_type} {computer.label}')
+    else:
+        echo.echo_report(f'  verdi -p {profile.name} computer configure {computer.transport_type} {computer.label}')
```

Comment:
Maybe for simplicity, we just always include the correct profile?
```suggestion
    echo.echo_report(f'  verdi -p {ctx.obj['profile']} computer configure {computer.transport_type} {computer.label}')
```

Reply:
I totally agree with that!! Changed.